### ISSUE TITLE
National flag emojis should not contain space character

### ISSUE THREAD

**Issue by mjwillson** [carpedm20/emoji#29]
So all the national flag emoji character combinations in your regexp and lookup tables, contain a space  in between the two "regional indicator" letters. This means they won't actually match the national flag sequences:

```python
In [7]: emoji.get_emoji_regexp().match("🇨🇦")

In [8]: emoji.get_emoji_regexp().match("🇨 🇦")
Out[8]: <_sre.SRE_Match object; span=(0, 3), match='🇨 🇦'>
```

**Comment by tbartelmess** [comment]
I've created a pull request for a fix #31

**Comment by carpedm20** [comment]
Thanks @tbartelmess
